test/e2e/cluster_test.go

```diff
@@ -582,6 +582,28 @@ var _ = Describe("Workload cluster creation", func() {
 		}, result)
 	})
 
+	It("Self manage NSG [PRBlocking]", func() {
```

Comment:
Lets make this not PRBlocking for now. Simply because setting up the external VCN might be a pain